### PR TITLE
Added examples

### PR DESCRIPTION
**Pull request by olligobber**

### CODE REVIEW

File: tests/example.at [hunk 1]
```diff
@@ -25,11 +25,13 @@ POSITIVE_TEST([cliquer],[example])
 POSITIVE_TEST([comp-parametrized-inherit],[example])
 POSITIVE_TEST([comp-parametrized-multilvl],[example])
 POSITIVE_TEST([comp-parametrized],[example])
+POSITIVE_TEST([counter],[example])
 POSITIVE_TEST([degree],[example])
 POSITIVE_TEST([dfa],[example])
 POSITIVE_TEST([dfa_live_vars],[example])
 POSITIVE_TEST([dfa_min],[example])
 POSITIVE_TEST([dfa_summary_function],[example])
+POSITIVE_TEST([dfa_parse],[example])
```
Comment (mmcgr) [inline tests/example.at]: Not quite alphabetical order

File: tests/example.at [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Souffle - A Datalog Compiler
-# Copyright (c) 2017, The Souffle Developers. All rights reserved.
+# Copyright (c) 2018, The Souffle Developers. All rights reserved.
```
Comment (mmcgr) [inline tests/example.at]: Don't update the original copyright year

Comment (olligobber) [reply tests/example.at]: Fixed in feb692275c6358a67396810e843d583af4e2f7ec

File: tests/example.at (hunk 1 above)
Comment (olligobber) [reply tests/example.at]: Fixed in feb692275c6358a67396810e843d583af4e2f7ec

**Pull request merged**